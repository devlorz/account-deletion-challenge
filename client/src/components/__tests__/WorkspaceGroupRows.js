import React from 'react'
import { render } from 'react-testing-library'

import WorkspaceGroupRows from '../WorkspaceGroupRows'

test('snapshot', () => {
  const { container } = render(
    <WorkspaceGroupRows
      workspaces={[
        {
          displayName: 'Lightning strike',
          spaceId: 'workspace1',
          transferableMembers: [
            {
              name: 'Ryan Lynch',
              _id: 'user2',
            },
            {
              name: 'Riker Lynch',
              _id: 'user3',
            },
            {
              name: 'Rydel Lynch',
              _id: 'user4',
            },
          ],
        },
      ]}
      groupTitle="The following workspaces require ownership transfer:"
      shouldDisplay={true}
    />
  )
  expect(container.firstChild).toMatchSnapshot()
})
