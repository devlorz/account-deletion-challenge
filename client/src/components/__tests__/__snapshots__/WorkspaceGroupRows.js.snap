// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  data-reactroot=""
>
  <h3>
    The following workspaces require ownership transfer:
  </h3>
  <div>
    <div
      style="margin-top: 1rem;"
    >
      <span>
        <!-- react-text: 6 -->
        Workspace: 
        <!-- /react-text -->
        <!-- react-text: 7 -->
        Lightning strike
        <!-- /react-text -->
      </span>
      <span />
    </div>
  </div>
</div>
`;
